<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-02T16:19:17 IST" name="Suite" finished-at="2024-09-02T16:19:17 IST" duration-ms="86">
    <groups>
    </groups>
    <test started-at="2024-09-02T16:19:17 IST" name="Test" finished-at="2024-09-02T16:19:17 IST" duration-ms="86">
      <class name="testcase.MyFirstTestFW">
        <test-method signature="LoginTest(java.lang.String,java.lang.String)[pri:0, instance:testcase.MyFirstTestFW@5890e879]" started-at="2024-09-02T16:19:17 IST" name="LoginTest" data-provider="bvtdata" finished-at="2024-09-02T16:19:17 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Some DataProvider public java.lang.String[][] utilities.ReadXLSData.getData(com.github.dockerjava.transport.DockerHttpClient$Request$Method) throws org.apache.poi.EncryptedDocumentException,java.io.IOException parameters unresolved:  at 0 type class com.github.dockerjava.transport.DockerHttpClient$Request$Method
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Some DataProvider public java.lang.String[][] utilities.ReadXLSData.getData(com.github.dockerjava.transport.DockerHttpClient$Request$Method) throws org.apache.poi.EncryptedDocumentException,java.io.IOException parameters unresolved:  at 0 type class com.github.dockerjava.transport.DockerHttpClient$Request$Method

at org.testng.internal.invokers.MethodInvocationHelper.getParameters(MethodInvocationHelper.java:239)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:158)
at org.testng.internal.Parameters.handleParameters(Parameters.java:809)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
      </class> <!-- testcase.MyFirstTestFW -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
